src/sss_iface/sbus_sss_client_async.c

```diff
@@ -2376,6 +2376,16 @@ sbus_emit_resp_negcache_ResetUsers
         "sssd.Responder.NegativeCache", "ResetUsers");
 }
 
+void
+sbus_emit_monitor_SetActive
```

Comment:
Why did this pop up..?

Reply:
Copy&paste leftover. Removed, nice catch.